readme.markdown

```diff
@@ -18,10 +18,9 @@ packages published to [npm](https://npmjs.org) that were originally intended for
 use in node but not browsers will work just fine in the browser too.
 
 Increasingly, people are publishing modules to npm which are intentionally
-designed to work in both node and in the browser using browserify and many
-packages on npm are intended for use in just the browser.
+designed to work in both node and in the browser. Using browserify
```

Comment:
Did you miss "and many packages on npm are intended for use in just the browser.?

Reply:
unfortunately I don't have the same context now as I did 3 years ago when I originally made this PR so I can't really say for sure if I missed that part. What I can say is I must have thought the sentence was confusing enough to warrant a helpful drive by PR. Thanks for getting to it.